vignettes/rblpapi-intro.Rmd

```diff
@@ -48,18 +48,79 @@ blpConnect()
 ```
 
 which will connect to the Bloomberg backend. Default values for the IP
-address (127.0.0.1) and port (8194) are used and can be overridden both as
-function arguments and via global options `blpHost` and `blpPort`.  Moreover,
-if option `blpAutoConnect` is set to `TRUE`, this connection is done at
-package load.  Additionally, if `blpAutoAuthenticate` is true, blpAuthenticate()
-will be called at package load.
+address (127.0.0.1) and port (8194) are used and can be overridden both
+as function arguments and via global options `blpHost` and `blpPort`.
+Moreover, if option `blpAutoConnect` is set to `TRUE`, this connection
+is done at package load.
 
-The resulting connection object is stored in a per-package environment providing
-a default value.  If needed it can be overridden in each accessor function.
+The resulting connection object is stored in a per-package environment
+providing a default value.  If needed it can be overridden in each
+accessor function.
 
 As the connection object is being held onto, the `blpDisconnect()`
-function is implemented as an empty function without side-effects.  The internal
-function `defaultConnection()` returns the current connection object.
+function is implemented as an empty function without side-effects.  The
+internal function `defaultConnection()` returns the default connection
+object.
+
+Default arguments and auto-connect can be set via `options()`
+
+```{R, eval=FALSE}
+options("blpAutoConnect" = TRUE)
+options("blpHost" = "x.x.x.x")
+options("blpPort" = 8194)
+options("blpAppName" = "yyy")
```

Comment:
Should we say that AppName is optional and used only for B-Pipe?   (If I have that right?)

Reply:
edited.  have a look.

Reply:
Thanks!